tensorflow_binding/setup.py

```diff
@@ -56,10 +57,31 @@
 warp_ctc_includes = [os.path.join(root_path, '../include')]
 include_dirs = tf_includes + warp_ctc_includes
 
-extra_compile_args = ['-std=c++11', '-fPIC']
+if tf.__version__ >= '1.4':
```

Comment:
I don't think it should be a string comparison. It doesn't work for version >= 1.10

Reply:
Sorry for that, I fixed the problem here #135

Reply:
Great!